admin/views/tab-permalinks.php

```diff
@@ -26,7 +26,7 @@
 $yform->checkbox( 'cleanreplytocom', __( 'Remove the <code>?replytocom</code> variables.', 'wordpress-seo' ) );
 
 /* translators %s expands to <code>.html</code> */
-echo '<p>' . sprintf( __( 'If you choose a permalink for your posts with %s, or anything else but a / on the end, this will force WordPress to add a trailing slash to non-post pages nonetheless.', 'wordpress-seo' ), '<code>.html</code>' ) . '</p>';
+echo '<p>' . sprintf( __( 'If you choose a permalink for your posts with %1$s, or anything else but a %2$s on the end, this will force WordPress to add a trailing slash to non-post pages nonetheless.', 'wordpress-seo' ), '<code>.html</code>', '<code>/</code>' ) . '</p>';
```

Comment:
`...anything else but a %2$s on the end...`
Should this be `on the end` or `at the end`?